prep-for-module-3-frontend.markdown

```diff
@@ -83,13 +86,13 @@ Please update your LinkedIn profiles with...
 
 Here are some additional resources:
 
-- Take a deeper dive into JSX with the [React docs](https://reactjs.org/docs/introducing-jsx.html) and this [tutorial](https://flaviocopes.com/jsx/)
-- Take a look at some extra [JSX gotchas (courtesy of tyler McGinnis)](https://ui.dev/jsx/)- [Testing with Fun Fun Function](https://www.youtube.com/playlist?list=PL0zVEGEvSaeF_zoW9o66wa_UCNE3a7BEr) (This is a playlist of 7 YouTube videos on testing)
+- Take a deeper dive into JSX with the [React docs](https://react.dev/learn/javascript-in-jsx-with-curly-braces) and this [tutorial](https://flaviocopes.com/jsx/)
```

Comment:
Can we change "tutorial" to "article"?  I was checking it out and it doesn't actually include a tutorial.